wolfcrypt/src/asn.c

```diff
@@ -104,6 +104,10 @@ ASN Options:
     #include <wolfssl/wolfcrypt/rsa.h>
 #endif
 
+#if defined(OPENSSL_EXTRA) || defined(OPENSSL_ALL) || defined(OPENSSL_EXTRA_X509_SMALL)
+    #include <wolfssl/openssl/ssl.h> /* for OBJ_sn2nid */
```

Comment:
Can we restructure this so that a crypto-level file (wolfcrypt/src/asn.c) is not including a SSL/TLS-level file (wolfssl/openssl/ssl.h)?  It's best if we don't create dependencies from the crypto level to the TLS level, just the other way around.

Reply:
This was for referring NIDs. Moved them to asn.h.